Pipfile

```diff
@@ -1,17 +1,27 @@
 [[source]]
-url = "https://pypi.python.org/simple"
-name = "pypi"
+url = "https://pypi.org/simple"
 verify_ssl = true
+name = "pypi"
 
 [packages]
-google-cloud-pubsub = "*"
+flask = "*"
 notifications-python-client = "*"
```

Comment:
sorry I'm not sure which line you are referring to?

Reply:
The `notifications-python-client` since we make the request directly.

Reply:
Ah actually I didn't see it was still used for the creation of the jwt. But we could use jwt directly.

Reply:
I'm happy to leave it in and use their JWT library?

Reply:
I don't mind. @LJBabbage @ajmaddaford thoughts?

Reply:
Is there a dependency cost to using it e.g. startup time?

Reply:
Only the time it takes to import the package, not the time to install it

Reply:
On my machine that time is:
CPU times: user 5 µs, sys: 0 ns, total: 5 µs
Wall time: 6.68 µs

Reply:
I think since we are installing a lib purely for 1 function, we should just have our own `create_jwt_token` wrapper and remove the dependency. We then don't have to tie into their package releases etc. I'm open on this, if we want to still keep it.

Reply:
In all honesty I'm just being lazy as it would mean extra code to test. Happy to take views @ajmaddaford & @LJBabbage

Reply:
Removed the Notify Python Client library